web/generate_site/templates/index-base.html

```diff
@@ -0,0 +1,161 @@
+    <div class="d-md-flex flex-md-equal w-75 my-md-3 mx-auto align-items-center">
+      <div class="col-md-5 p-lg-1 mx-auto my-5">
+        <img style="width: 60%;" src="img/banner.png" />
+      </div>
+      <div class="col-md-5 p-lg-3 mx-auto my-5">
+        <p class="lead fw-normal">
+          "The more I study, the more insatiable do I feel my genius for it to be."
+        </p>
+        <h4>Ada Lovelace</h4>
+        <h1 class="pink display-4 fw-normal">PyLadiesCon</h1>
+        <h4 class="gradient special-font">2023 Online</h4>
+        <p class="lead fw-normal">
+          1-3 Dec | Multi-language | Multi-timezone
+        </p>
+        <div class="w-50 mx-auto">
+        <a class="btn btn-outline-secondary btn-pink-gradient" href="https://pretalx.com/pyladiescon-2023/cfp" target="_blank">Call for proposal</a>
+        </div>
+        <div class="social w-50 mx-auto pt-3">
+          <a href="https://fosstodon.org/@pyladiescon"><i class="fab fa-mastodon fa-2x px-2"></i></a>
+          <a href="https://twitter.com/pyladiescon"><i class="fab fa-twitter fa-2x px-2"></i></a>
+          <a href="https://instagram.com/pyladiescon"><i class="fab fa-instagram fa-2x px-2"></i></a>
+          <a href="https://www.linkedin.com/company/pyladiescon"><i class="fab fa-linkedin fa-2x px-2"></i></a>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <!-- about -->
+  <div class="d-md-flex flex-md-equal w-100 my-md-3 ps-md-3">
+    <div class="bg-light me-md-3 py-3 px-3 py-md-5 px-md-5 overflow-hidden">
+      <div class="my-3 py-3 px-5">
+        <img class="icon-section" src="img/about-icon.png"/>
+        <h3 class="display-6 special-font pink">About the conference</h2>
+        <p class="lead">
+          Attention all PyLadies community members! We’re excited
+          to share that we are in the early stages of planning a PyLadies
+          Conference (PyLadiesCon), a transformative event designed to promote
+          diversity, learning, and empowerment within the Python community.
+          🎉
+        </p>
+        <p class="lead">
+          Save the date! The conference will take place on the first weekend
+          of December, where we’ll gather together for a weekend filled with
+          insightful talks, engaging panels, and collaborative networking
+          opportunities.
+        </p>
+        <p class="lead">
+          <a class="rarrow-link text-black" href="#">Read More</a>
+        </p>
+        <p class="lead">
+          <a class="rarrow-link text-black" href="#">Code of Conduct</a>
+        </p>
+      </div>
+    </div>
+  </div>
+
+  <!-- registration details -->
+  <div class="d-md-flex flex-md-equal w-100 my-md-3 ps-md-3">
+    <div class="bg-light me-md-3 py-3 px-3 py-md-5 px-md-5 overflow-hidden">
+      <div class="my-3 py-3 px-5">
+        <img class="icon-section" src="img/details-icon.png"/>
+        <h3 class="display-6 special-font pink">Registration Details</h2>
```

Comment:
```suggestion
        <h3 class="display-6 special-font pink">Join us!</h2>
```
This might be a placeholder for registration, but since we don't have the registration yet, the section heading seems misleading. I suggested a different wording, and we can add the "Registration Details" later once we have it.